multi/imxrt-multi/cm4.c

```diff
@@ -123,14 +127,17 @@ static inline int fifo_pushPacket(packet_t *packet, fifo_t *fifo)
 {
 	int i, size;
 
-	if (packet->byte.bno > sizeof(packet->byte.payload))
+	if (packet->byte.bno > sizeof(packet->byte.payload)) {
 		size = sizeof(packet->byte.payload);
-	else
+	}
+	else {
 		size = packet->byte.bno;
+	}
 
 	for (i = 0; i < size; ++i) {
-		if (!fifo_push(packet->byte.payload[i], fifo))
+		if (!fifo_push(packet->byte.payload[i], fifo)) {
```

Comment:
```suggestion
		if (fifo_push(packet->byte.payload[i], fifo) == 0) {
```
because that PR changes the style later in the code anyway.